webapp/packages/core-blocks/src/ResourcesHooks/useResource.ts

```diff
@@ -470,7 +470,7 @@ export function useResource<
   useEffect(() => {
     refObj.use(loadKey);
     if (canLoad && !result.isError()) {
-      result.load();
+      result.load().catch(() => ({}));
```

Comment:
```suggestion
      result.load();
```
not needed, result.load have try catch inside